boards/arm/bcm958402m2_m7/bcm958402m2_m7_defconfig

```diff
@@ -18,3 +18,9 @@ CONFIG_UART_NS16550=y
 # Enable Console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+# Enable PAXDMA
+CONFIG_DMA=y
+CONFIG_DMA_64BIT=y
```

Comment:
Should probably also remove DMA_64BIT as its not used by the PAXDMA driver.

Reply:
DMA_64BIT is needed to enable passing 64 bit source and destination buffer to the DMA driver as in our use case we need 64bit addresses.

Reply:
gotcha.